app/cmd/plugin_search.go

```diff
@@ -71,19 +71,69 @@ func searchPlugins(plugins *client.AvailablePluginList, keyword string) []client
 	return result
 }
 
-// Output output the data into buffer
+func showVersionPlugins(plugins []client.AvailablePlugin) (p []client.CenterPlugin) {
```

Comment:
I believe that you want to re-organize the data instead of showing the version. Please consider changing this method name.